index.html

```diff
@@ -1,10 +1,131 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-	<meta charset="UTF-8">
-	<title>Document</title>
-<link rel="stylesheet" href="/assets/style.css" /></head>
-<body>
-	<h1>Hi</h1>
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <title>Document</title>
+    <link rel="stylesheet" type="text/css" href="scss/vendor/reset.css" />
+
+    <link
+      href="https://fonts.googleapis.com/css?family=Raleway&display=swap"
+      rel="stylesheet"
+    />
+
+    
+  <link rel="stylesheet" href="/assets/style.css" /></head>
+  <body>
+    <div class="container">
+      <div class="start-screen-block">
+        <header>
+          <ul>
+            <li><a href="#">home</a></li>
+            <li><a href="#">portfolio</a></li>
+            <li><a href="#">blog</a></li>
+            <li><a href="#">pages</a></li>
+            <li><a href="#">about us</a></li>
+            <li><a href="#">contact</a></li>
+          </ul>
+        </header>
+        <div class="header-logo">
+          <img src="src/img/logo.png" />
+          <p>Expire</p>
+        </div>
+
+        <div class="header-slider">
+          <div class="arrows">
+            <img src="src/img/left.png" class="left-arrow" />
+            <img src="src/img/right.png" class="right-arrow" />
+          </div>
+          <p class="slider-headline">
+            Expire
+          </p>
+          <img src="src/img/line.png" class="line" />
+          <p class="slider-paragraph">
+            Professionaly designed, carefully made for your enjoyement
+          </p>
+          <div class="slider-buttons">
+            <button class="explore-btn">explore</button>
+            <button class="learn-btn">
+              learn more
+            </button>
+          </div>
+          <div class="slider-dots">
+            <div class="focused"></div>
```

Comment:
It's better to use some common class for all dots with common styles like `width, height, etc.`, for example `slider-dots-item`, and some modificator for focused state, for example `slider-dots-item-focused`

Reply:
Done